### PR TITLE
Switch from `heim` lib to `sysinfo` for working with processes

### PR DESCRIPTION
**Pull request by loony-bean**
Heim appears to be not actively maintained any more.

### CODE REVIEW

File: src/machine.rs [hunk 1]
```diff
@@ -238,8 +240,21 @@ impl<'m> Machine<'m> {
     pub async fn stop(&mut self) -> Result<(), Error> {
         let vm_id = self.config.vm_id();
         trace!("{vm_id}: Killing VM...");
-        let child = heim::process::get(self.pid.try_into()?).await?;
-        child.kill().await?;
+
+        let pid = self.pid;
+        let killed = task::spawn_blocking(move || {
+            let sys = System::new_all();
```
Comment (GuillaumeGomez) [inline src/machine.rs:246]: If you only care about one process to check if it's alive, better do this:

```suggestion
            let sys = System::new();
            sys.refresh_process_specifics(ProcessRefreshKind::new());
```

File: src/machine.rs [hunk 2]
```diff
@@ -238,8 +240,21 @@ impl<'m> Machine<'m> {
...
+            match sys.process(Pid::from(pid)) {
+                Some(process) => Ok(process.kill()),
+                None => Err(Error::ProcessNotRunning(pid)),
+            }
```
Comment (GuillaumeGomez) [inline src/machine.rs:250]: If you only care about one process to check if it's alive, better do this:

```suggestion
            let sys = System::new();
            if sys.refresh_process_specifics(Pid::from(pid), ProcessRefreshKind::new()) {
                match sys.process(Pid::from(pid)) {
                    Some(process) => Ok(process.kill()),
                    None => Err(Error::ProcessNotRunning(pid)),
                }
            }
```

Comment (zeenix) [reply src/machine.rs:250]: Thanks @GuillaumeGomez. A bit late for accepting the suggestion here so I'll create a commit with your name on it, unless you want to submit a PR? ;)

Comment (GuillaumeGomez) [reply src/machine.rs:250]: I'll submit a PR then. ;)

**Pull request merged**